arm-compute/compositeComputeClient.json

```diff
@@ -4,7 +4,7 @@
     "description": "Composite Swagger for Compute Client"
   },
   "documents": [
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/2016-03-30/swagger/compute.json",
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/2015-11-01-preview/swagger/containerService.json"
+    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/dev/arm-compute/2016-03-30/swagger/compute.json",
```

Comment:
Good catch!! It's much better to use relative path, in order to avoid this type of silly mistakes... :-(